src/Storages/StorageReplicatedMergeTree.h

```diff
@@ -263,6 +274,10 @@ class StorageReplicatedMergeTree final : public shared_ptr_helper<StorageReplica
 
     bool createEmptyPartInsteadOfLost(zkutil::ZooKeeperPtr zookeeper, const String & lost_part_name);
 
+    String getZooKeeperName() const { return zookeeper_name; }
```

Comment:
Add a comment. What is it? Name of the table in ZooKeeper? Or name of ZooKeeper cluster itself?

Reply:
Table can use non-default zookeeper cluster (https://github.com/ClickHouse/ClickHouse/blob/master/src/Storages/StorageReplicatedMergeTree.cpp#L215).
It confuses me a bit, but looks like someone can use this ability with different zookeepers for different tables.